code/modules/power/supermatter/supermatter.dm

```diff
@@ -65,6 +65,7 @@
 #define SUPERMATTER_WARNING_PERCENT 100
 
 #define SUPERMATTER_COUNTDOWN_TIME 30 SECONDS
+#define SUPERMATTER_EMAG_SPACE_DAMAGE 4 //higher for faster delamination when emagged and spaced. 4 is around best as it has roughly the same speed as a high temperature delam
```

Comment:
Oh god no, the max sm damage is given by `(DAMAGE_HARDCAP * explosion_point)`, which is 900 * 0.002, or 1.8 each tick, 3.6 per second.

Reply:
Spacing the engine is the braindead and easy solution I intend to punish hard for lack of creativity, but since a hardcap exists might as well use it. Thanks I didn't spot this.

Reply:
Damage now complies with hardcap.